### PR TITLE
OBSDOCS-607: Update minor error in callout

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-607

Link to docs preview:
https://68698--docspreview.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-systemd#cluster-logging-systemd-scaling_cluster-logging-systemd

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kabirbhartiRH** [comment]
/lgtm.

One very minor suggestion would be to add **'volatile' to store logs in memory in '/run/log/journal/' (lost on reboot)**

**Comment by kabirbhartiRH** [comment]
/lgtm

One very minor suggestion would be to add 'volatile' to store logs in memory in '/run/log/journal/' (lost on reboot)

File: modules/cluster-logging-systemd-scaling.adoc [hunk 1]
```diff
@@ -65,7 +65,7 @@ Specify `yes` to compress the message or `no` to not compress. The default is `y
 include a unit: "year", "month", "week", "day", "h" or "m". Enter `0` to disable. The default is `1month`.
 <5> Configure rate limiting. If more logs are received than what is specified in `RateLimitBurst` during the time interval defined by `RateLimitIntervalSec`, all further messages within the interval are dropped until the interval is over. It is recommended to set `RateLimitIntervalSec=30s` and `RateLimitBurst=10000`, which are the defaults.
 <6> Specify how logs are stored. The default is `persistent`:
-* `volatile` to store logs in memory in `/var/log/journal/`.
+* `volatile` to store logs in memory in `/run/log/journal/`.
```
Comment (abrennan89) [inline modules/cluster-logging-systemd-scaling.adoc:68]:
```suggestion
* `volatile` to store logs in memory in `/run/log/journal/`. These logs are lost after rebooting.
```
Do you mean something like this @kabirbhartiRH ?

Comment (kabirbhartiRH) [reply modules/cluster-logging-systemd-scaling.adoc:68]: @abrennan89 Yes :)

**Pull request merged**